save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DiktatDemoResult.kt

```diff
@@ -0,0 +1,17 @@
+package com.saveourtool.save.demo.diktat
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Class that represents the output of diktat demo
+ *
+ * @param warnings list of warnings that were found
+ * @param outputText resulting file text
+ * @property warnings
+ * @property outputText
+ */
+@Serializable
+data class DiktatDemoResult(
```

Comment:
Same question

Reply:
The output can be unified as far as all the tools surely have result, will add `DemoResult` interface

Reply:
Done